edx_solutions_api_integration/courses/views.py

```diff
@@ -2015,6 +2023,110 @@ def get(self, request, course_id):  # pylint: disable=W0613,W0221
         return Response(data, status=status.HTTP_200_OK)
 
 
+class CompletionList(SecureListAPIView):  # pylint: disable=too-many-ancestors
+    """
+    ### The CompletionList allows clients to view user's course module completion entities
+    to monitor a user's progression throughout the duration of a course,
+    - URI: ```/api/courses/{course_id}/completions```
+    - GET: Returns a JSON representation of the course, content and user and timestamps
+    - GET Example:
+        {
+            "count":"1",
+            "num_pages": "1",
+            "previous": null
+            "next": null
+            "results": [
+                {
+                    "id": 2,
+                    "user_id": "3",
+                    "course_id": "32fgdf",
+                    "content_id": "324dfgd",
+                    "stage": None,
+                    "created": "2014-06-10T13:14:49.878Z",
+                    "modified": "2014-06-10T13:14:49.914Z"
```

Comment:
@jcdyer, the response I got while testing this in my devstack looked something like

```json
{
"id": 18,
"user_id": 5,
"content_id": "block-v1:edX+DemoX+Demo_Course+type@poll+block@44cc060f582743899d70b850097cf029",
"course_id": "course-v1:edX+DemoX+Demo_Course",
"stage": null,
"created": "2018-10-26T18:22:59.416943Z",
"modified": "2018-10-26T18:22:59.417711Z",
"course_key": "course-v1:edX+DemoX+Demo_Course",
"block_key": "block-v1:edX+DemoX+Demo_Course+type@poll+block@44cc060f582743899d70b850097cf029",
"block_type": "poll",
"completion": 1,
"user": 5
}
```
so this example response has to be updated.

Reply:
Updated.